git_machete/client.py

```diff
@@ -1060,14 +1077,16 @@ def print_line_prefix(branch_: LocalBranchShortName, suffix: str) -> None:
                              f"or that some extra branch should be added between each of these branches and its parent"
 
             if not opt_list_commits:
-                second_part = "Run `git machete status --list-commits` or `git machete status --list-commits-with-hashes` to see more details"
+                second_part = "Run `git machete status --list-commits` or " \
+                              "`git machete status --list-commits-with-hashes` to see more details"
             elif len(branches_in_sync_but_fork_point_off) == 1:
-                second_part = f"Consider using `git machete fork-point --override-to=<revision>|--override-to-inferred|--override-to-parent" \
-                              f" {branches_in_sync_but_fork_point_off[0]}`,\nor reattaching `{branches_in_sync_but_fork_point_off[0]}` " \
-                              f"under a different parent branch"
+                second_part = f"Consider using `git machete fork-point --override-to=<revision>|--override-to-inferred" \
+                              f"|--override-to-parent {branches_in_sync_but_fork_point_off[0]}`,\nor reattaching " \
```

Comment:
Whoops 

```
+ prohibit-strings-split-without-delimiter.sh
git_machete/client.py:1083-1084: it looks that the string is split without a delimiter, words will be glued up together when printed

                second_part = f"Consider using `git machete fork-point --override-to=<revision>|--override-to-inferred" \
                              f"|--override-to-parent {branches_in_sync_but_fork_point_off[0]}`,\nor reattaching " \


git_machete/client.py:1087-1088: it looks that the string is split without a delimiter, words will be glued up together when printed

                second_part = "Consider using `git machete fork-point --override-to=<revision>|--override-to-inferred" \
                              "|--override-to-parent <branch>` for each affected branch,\n" \
```

Reply:
I mean... this check (of my authorship 😅) is very useful in general, but here it's clearly a false positive... pls maybe move the point of line break so that it doesn't trigger here

Reply:
Yeah, already did